test/bst.t

```diff
@@ -134,9 +134,8 @@ Testing --limit-core / general tests
 	bst: error in --limit-core value: Invalid argument
 	[1]
 
-	$ bst --limit-core=0xffffffffffffffffffffffffe
+	$ bst --limit-core=0xffffffffffffffffffffffffe 2>&1 | sed -e 's/Result not representable/Numerical result out of range/'
 	bst: error in --limit-core value: Numerical result out of range
-	[1]
```

Comment:
Was this just a WIP / testing change?  Ah, it's actually for handling different systems.  I guess that makes sense.

Reply:
Yeah, alpine is annoying here. Plus since I can't assume the shell supports -o pipefail (on my laptop, /bin/sh doesn't), it also swallows the exit status...